src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -1321,6 +1322,39 @@ class FlowCrudSpec extends HealthCheckSpecification {
         !error && flowHelperV2.deleteFlow(flow.flowId)
     }
 
+    @Tidy
+    def "Unable to update to a flow with incorrect maxLatency input - #title"() {
+        given: "A flow"
+        def (Switch srcSwitch, Switch dstSwitch) = topology.activeSwitches
+        def flow = flowHelperV2.randomFlow(srcSwitch, dstSwitch)
+        flowHelperV2.addFlow(flow)
+
+        when: "Try to update the flow "
+        def flowInfo = northboundV2.getFlow(flow.flowId)
```

Comment:
Calling Northbound is quite 'heavy' operation (it takes seconds). You already have `FlowRequestV2` object in `flow` variable, reuse it.

Reply:
replaced